src/app/containers/CpsFeaturesAnalysis/index.jsx

```diff
@@ -23,20 +23,41 @@ const PromoListComponent = ({ promoItems, dir }) => {
   const viewRef = useViewTracker(eventTrackingData.block);
   const { isAmp } = useContext(RequestContext);
 
-  const StoryPromo = isAmp || isLive() ? _StoryPromo : FrostedGlassPromo;
+  const promoVariation = useOptimizelyVariation(
+    'high_impact_feature_analysis_promo',
+  );
+
+  let hasFrostedGlassPromo = false;
+
+  if (promoVariation) {
+    if (promoVariation === 'variation_1' && !isAmp) {
+      hasFrostedGlassPromo = true;
+    }
+  }
 
   return (
     <StoryPromoUl>
-      {promoItems.map(item => (
+      {promoItems.map((item, i) => (
         <StoryPromoLi key={item.id || item.uri} ref={viewRef}>
-          <StoryPromo
-            item={item}
-            dir={dir}
-            displayImage
-            displaySummary={false}
-            serviceDatetimeLocale={serviceDatetimeLocale}
-            eventTrackingData={eventTrackingData}
-          />
+          {i === 0 && hasFrostedGlassPromo ? (
```

Comment:
`isFirstPromo` or `isFirstItem` might be clearer here

Reply:
`hasFrostedGlassPromo` should maybe have `is` prefix instead? maybe `isHighImpactVariation` is a more suitable name?

Reply:
Agreed. Done :)